src/plugin/settings/engine.cpp

```diff
@@ -267,21 +272,25 @@ Result<const ISettingsResponse *> Engine::settingsReceived()
         {
 #if defined(__linux__)
             // On Linux we need to create the folder if it does not yet exist
+            NX_PRINT << "creating the folder if it does not yet exist" << std::endl;
             if (!fs::create_directory(mountDir, errCode))
             {
+                NX_PRINT << "Unable to create mount directory with error: " + errCode.message() << std::endl;
                 return error(ErrorCode::internalError,
                              "Unable to create mount directory with error: " + errCode.message());
             }
             fs::permissions(mountDir, fs::perms::all, fs::perm_options::add, errCode);
             if (errCode)
             {
+                NX_PRINT <<  "Unable to set mount directory permissions with error: " + errCode.message() << std::endl;
                 return error(ErrorCode::internalError,
                              "Unable to set mount directory permissions with error: " + errCode.message());
             }
 #endif
         }
 
         // Create file cache if it does not exist
+        NX_PRINT << "creating file cache if it does not exist" << std::endl;
```

Comment:
Instead of this print, we could just print at the top of the else block that the file cache directory doesn't exist (and will be created).